scripts/globals/regimes.lua

```diff
@@ -987,6 +994,7 @@ end
 dsp.regime.bookOnEventUpdate = function(player, option, regimeType)
     local pRegime = player:getVar("fov_regimeid")
     local zoneId = player:getZoneID()
+    local page = getPageByRegimeId(regimeType, player:getVar("fov_zoneid"), pRegime)
```

Comment:
This is still going to die on the regimeType if you have FoV active and try to review it on a GoV

Reply:
good catch. updated again.